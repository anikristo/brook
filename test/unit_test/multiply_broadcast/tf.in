2 9
4 5 6 2 6 -3 -5 4 -22
5 6 7 3 7 -2 -4 5 -21