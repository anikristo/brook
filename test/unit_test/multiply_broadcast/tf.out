2 9
8.0 10.0 12.0 4.0 12.0 -6.0 -10.0 8.0 -44.0
10.0 12.0 14.0 6.0 14.0 -4.0 -8.0 10.0 -42.0
